### DISCUSSION TITLE
this site cant be reached

### DISCUSSION THREAD

**Discussion by Kryxzort** [community/community#56286] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

any raw format i open, dont load and when i directly try, it says this site cant be reached. cant even see raw format of mine

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @KGFKILLERPRO ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by Kryxzort** [reply]
okay thx

**Comment by Kryxzort** [comment]
helpp
